fa25-team-a/analysis/Property_Price_Tiers.ipynb

```
@@ -0,0 +1,167 @@
+        "\n",
+        "**Dataset**: full_merged_V1.csv (https://drive.google.com/file/d/1Pp6nDx-vK-6rEFez_5gLpmtZr59BLhqA/view?usp=drive_link)\n",
+        "\n",
+        "**Instruction**: Download the CSV file above and move it to the ds-chapa-affordable-housing/fa25-team-a/data folder, then run the cells below\n",
+        "\n",
+        "This notebook performs the following tasks on the full merged CHAPA resale dataset:\n",
+        "\n",
+        "1. Drop rows with missing `Town` or `Address`.\n",
+        "2. Clean and convert `Maximum Resale Price` to numeric.\n",
+        "3. Fix encoding issues in `income_bracket` and `asset_bracket`.\n",
+        "4. Identify and remove invalid or zero resale prices.\n",
+        "5. Create 4-tier price categories based on quartiles.\n",
```

Ruff (0.14.8)

String contains ambiguous `‚` (SINGLE LOW-9 QUOTATION MARK). Did you mean `,` (COMMA)?

(RUF001)